.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.1.5
```

Comment:
In line with above, we'll both be on the same ruby. How do I learn to figure out which Ruby version works best?